### PR TITLE
Update code owners

### PR DESCRIPTION
**Pull request by madebyrogal**
## Description

Changes proposed in this pull request:
- add my (madebyrogal) as a new code owner

## Related issue(s)

None

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # BotKube maintainers (email: dev-team@botkube.io)
 # Maintainers are listed alphabetically
 
-* @ezodude @mszostok @pkosiec @PrasadG193
+* @madebyrogal @ezodude @mszostok @pkosiec @PrasadG193
```
Comment (mszostok) [inline .github/CODEOWNERS:4]: **nit** as stated above "# Maintainers are listed alphabetically", so please update the list :) 

```suggestion
* @ezodude @madebyrogal @mszostok @pkosiec @PrasadG193
```

**Review by mszostok — APPROVED** [review]
Welcome to BotKube team! 🎉

please apply small suggestion before merging :)

**Review by pkosiec — APPROVED** [review]
Welcome 🥳 Please see Mateusz's comment before merging.

File: .github/CODEOWNERS (hunk 1 above)
Comment (madebyrogal) [reply .github/CODEOWNERS]: Done. Thanks

**Review by pkosiec — APPROVED** [review]
👌

**Review by mszostok — APPROVED** [review]
don't worry about CI, it's just about 429 from GitHub, can be ignored for this PR👍

**Pull request merged**